customisation/adaptive.md

```diff
@@ -11,12 +11,23 @@ Adaptive machine translation is applicable to [post-editing](../workflows/post-e
 In adaptive machine translation, the system is customised while the human post-editor fixes the machine translation output, instead of after batch retraining.
 
 Adaptive machine translation is an example of online machine learning and human-in-the-loop (HITL).
-
-### Companies
-
-In 2019, ModernMT launched adaptive machine translation.
-
-In 2020, Amazon Translate launched Active Custom Translation, an adaptive machine translation feature.
+### API support
+
+Adaptive machine translation first became practical with the rise of [neural machine translation](/approaches/neural-machine-translation.md).  [Lilt](/apis/lilt.md) launched the first adaptive feature in 2017.
```

Comment:
According to our [Style Guide](https://machinetranslate.org/style#spacing), we should separate sentences with a single line break.